CHANGELOG.md

```diff
@@ -27,3 +27,4 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add slugs to product/category/collection/page translations. Allow to query by translated slug - #16449 by @delemeator
 - Fixed a crash when the Decimal scalar is passed a non-normal value - #16520 by @patrys
 - Fixed a bug when saving webhook payload to Azure Storage - #16585 by @delemeator
+- Fixed bug when manual line discount doesn't override line-level vouchers - #16738 by @zedzior
```

Comment:
Should it be part of `breaking changes`? 
If yes, I think we should add at least one example of how the discount behavior will change.

Reply:
Thanks for the remark. I would say this was a temporary bug rather than breaking change. We have never documented that manual line discount can be combined with other discounts. Since we introduced `sales`, the manual line discount has overridden other discounts. This will be documented in upcoming `Price Calculation` section: https://github.com/saleor/saleor-docs/pull/1328/files